CIP-0068/README.md

```diff
@@ -0,0 +1,214 @@
+
+## Motivation
+
+[CIP-0025](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0025) has plenty of limitations, to name a few
+
+- zero programmability
+- it is very hard to update/evolve the metadata
+- the metadata are not readable from a plutus validator. 
+
+This standard tackles all these problems and offers many more advantages, not only for NFTs, but also for any asset class that may follow. Additionally, this CIP will introduce a way to classify tokens so that third parties like wallets can easily know what the kind of token it is.
+
+## Considerations
+
+The basic idea is to have two assets issued, where one references the other. We call these two a `reference NFT` and an `user token`, where the `user token` can be an NFT, FT or any other asset class that is transferable and represents any value. So, the `user token` is the actual asset that lives in a user's wallet.
+
+To find the metadata for the `user token` you need to look for the output, where the `reference NFT` is locked in. How this is done concretely will become clear below. Moreover, this output contains a datum, which holds the metadata. The advantage of this approach is that the issuer of the assets can decide how the transaction output with the `reference NFT` is locked and further handled. If the issuer wants complete immutable metadata, the `reference NFT` can be locked at the address of an unspendable script. Similarly, if the issuer wants the NFTs/FTs to evolve or wants a mechanism to update the metadata, the `reference NFT` can be locked at the address of a script with arbitrary logic that the issuer decides.
+
+Lastly and most importantly, with this construction, the metadata can be used by a PlutusV2 script with the use of reference inputs [CIP-0031](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0031). This will drive further innovation in the token space. 
+
+## Specification
+
+This is the registered `asset_name_label` value
+
+| asset_name_label            | class        | description                                               |
+| --------------------------- | ------------ | --------------------------------------------------------- |
+| 100                         | NFT          | Reference NFT locked at a script containing the datum     |
+
+For a correct relationship between the `user token` and the `reference NFT` a few conditions **must** be met.
+- The `user token` and `reference NFT` **must** be under the same policy ID. 
+- For a specific `user token` there **must** exist exactly **one** `reference NFT`
+- The `user token` and associated `reference NFT` **must** follow the standard naming pattern. The asset name of both assets is prefixed with its respective `asset_name_label` followed by a pattern defined by the asset class (e.g. asset_name_label 222)
+
+Some remarks about the above,
+1. The `user token` and `reference NFT` do not need to be minted in the same transaction. The order of minting is also not important.
```

Comment:
This is a clever approach that also doesn't depend on any logging standard. 

One way you could get around some of the restrictions with your approach is to instead having some meta-NFT for the project that is used in all the transactions that create the reference NFTs and the user tokens. This would avoid the asset-name stuffing issue your CIP introduces and this meta-NFT could be kept after the initial mint is done to add other reference NFT CIPs for the collection in the future without having to keep the policy used for the user tokens open. This is of course significantly more complicated than your approach though

Reply:
I was thinking more something similar to what NMKR is doing with their DID proposal (https://github.com/cardano-foundation/CIPs/pull/294) except instead of an open minting policy to store multiple DIDs, the open minting policy is what is connected to other minting scripts

In other words, the minting policy script in the your image servers two purposes: minting the assets and connecting to the reference scripts. This could be split up into two different scripts similar to the diagram in the DID proposal